package.json

```diff
@@ -94,8 +100,8 @@
     ]
   },
   "workspaces": [
-    "src/implementations/react/components/*",
```

Comment:
Please leave react implementation in the workspace and in the code base for the moment and consider the ecl website update out of scope.